code/drv_can/src/can_sniffer/can_sniffer.py

```diff
@@ -61,37 +66,45 @@ def __init__(self, addr : int, size : int, data : int | bytearray) -> None:
         '''
         self.addr = addr
         self.dlc = size
-        if self.dlc > _MAX_DLC_SIZE:
+        if self.dlc > _Constants.MAX_DLC_SIZE:
             log.error(f"Message payload size on bytes (size = {self.dlc}) \
-                      is higher than {_MAX_DLC_SIZE}")
+                      is higher than {_Constants.MAX_DLC_SIZE}")
             raise BytesWarning("To many element for a CAN message")
-        if isinstance(data,int):
-            self.data = data.to_bytes(size, byteorder='little', signed = False)
+        if isinstance(payload,int):
+            self.payload = payload.to_bytes(size, byteorder='little', signed = False)
         else:
-            self.data = data
+            self.payload = payload
 
-class DrvCanFilterC():
+class DrvCanFilterC:
```

Comment:
Create another class (`_CanActiveFilterC`) applying inheritance. It must contains match and close_chan functions

Reply:
Done